test/es7/asyncawait-functionality.js

```diff
@@ -675,6 +676,85 @@ var tests = [
             });  
         }
     },
+    {
```

Comment:
I forget, have you run these tests with force defer parse?  Curious to know if deferred parse is truly now a moot issue.